From a48293ed75f8ba0a05508f9d9ccf77096b554f63 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 14 Jul 2019 13:00:44 +0200
Subject: [PATCH 1/1] efi_loader: fix function comments in cmd/bootefi.c

Describe efi_set_bootdev().

Change function descriptions to Sphinx style.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/bootefi.c | 16 +++++++++++++---
 1 file changed, 13 insertions(+), 3 deletions(-)

diff --git a/cmd/bootefi.c b/cmd/bootefi.c
index 04d3e3e4a7..a45bfd139f 100644
--- a/cmd/bootefi.c
+++ b/cmd/bootefi.c
@@ -24,7 +24,7 @@ DECLARE_GLOBAL_DATA_PTR;
 static struct efi_device_path *bootefi_image_path;
 static struct efi_device_path *bootefi_device_path;
 
-/*
+/**
  * Set the load options of an image from an environment variable.
  *
  * @handle:	the image handle
@@ -143,7 +143,7 @@ done:
 	return ret;
 }
 
-/*
+/**
  * efi_carve_out_dt_rsv() - Carve out DT reserved memory ranges
  *
  * The mem_rsv entries of the FDT are added to the memory map. Any failures are
@@ -342,7 +342,7 @@ static int do_efibootmgr(void)
 	return CMD_RET_SUCCESS;
 }
 
-/*
+/**
  * do_bootefi_image() - execute EFI binary
  *
  * Set up memory image for the binary to be loaded, prepare device path, and
@@ -612,6 +612,16 @@ U_BOOT_CMD(
 	bootefi_help_text
 );
 
+/**
+ * efi_set_bootdev() - set boot device
+ *
+ * This function is called when a file is loaded, e.g. via the 'load' command.
+ * We use the path to this file to inform the UEFI binary about the boot device.
+ *
+ * @dev:	device, e.g. "MMC"
+ * @devnr:	number of the device, e.g. "1:2"
+ * @path:	path to file loaded
+ */
 void efi_set_bootdev(const char *dev, const char *devnr, const char *path)
 {
 	struct efi_device_path *device, *image;
-- 
2.20.1

